Walking into your bluespace ghost gives you damage first

CSCMe opened the pull request:
Alternative to #30721 
Gives you increasing amounts of damage when you walk into your own ghost, warning you about what happens if you do it too often.
The damage is most noticeable and knocks you out on the touch before death.

I think that there should still be a "punishment" for messing with bluespace too much.

This should handle all the gripes GeneralIcamo had with bluespace deaths

* Not a newby trap because it warns you now
* Not _that_ much of a easy-out for traitors
* It's no longer an accident if you die of bluespace (it's also a proper reason to get bwoinked and asked why you did it now)

🆑 CSCMe
tweak: Touching your bluespace ghost too often now gives you damage before you die
/🆑

SierraKomodo commented on code/game/gamemodes/endgame/bluespace_jump/bluespace_jump.dm:
```diff
@@ -104,13 +104,22 @@
+			brain.take_internal_damage(round(reality * 1.5))
 		else if(reality > 3)
-			to_chat(daddy, "<span class='danger'>Something is definitely wrong. Why do you think YOU are the original?</span>")
+			to_chat(daddy, SPAN_DANGER("This is weird. Which one of you two was original again..?"))
```
```suggestion
			to_chat(daddy, SPAN_DANGER("This is weird. Which one of you two was the original again..?"))
```

SierraKomodo commented on code/game/gamemodes/endgame/bluespace_jump/bluespace_jump.dm:
```diff
@@ -104,13 +104,22 @@
 		reality++
-		if(reality > 5)
-			to_chat(daddy, "<span class='notice'>Yep, it's certainly the other one. Your existance was a glitch, and it's finally being mended...</span>")
+		var/obj/item/organ/internal/brain/brain = daddy.internal_organs_by_name[BP_BRAIN]
```
This will need a sanity check for if brain actually exists, and testing for how this interacts with non-human mobs like borgs, simple mobs, etc.

CSCMe replied:
non-humans, borgs and simple mobs just don't take any damage on the brain-damage stage
but they do take damage on the final one

SierraKomodo reviewed (changes requested):
The fact the var is `daddy` bothers me.

Spookerton commented:
This does not seem favorable to either alternative, since it effectively only makes the current model also spurt blood.

I'm unconvinced there is an issue here to *solve* re #30728 and #30721, considering people have been getting along happily with the existing mechanic for four entire years until *yesterday*.

CSCMe commented:
So the current bluespace ghost system is _extremely_ broken.
Even if you're against removing and changing the way it works, it needs a fix.
I seriously can't believe that nobody encountered the errors I found while testing the current system

CSCMe committed:
Walking into your bluespace ghost now deals damage before death

Now requires 7 touches instead of 6 to die

Take minor brain damage from touch 4 and overall damage on touch 6

CSCMe commented:
Right, seeing as this one is slowly but surely getting the same reception as the removal (with ~the same amount of people being for and against) imma just use this for the damage/death changes and put the fixes in a new PR

Ready for review

Spookerton commented:
pass, will discuss the other
